README.md

```diff
@@ -77,6 +77,7 @@ before_install:
 | -r   | --report          | Shows the `npm audit --json` report (default `true`)                              |
 | -a   | --advisories      | Vulnerable advisory ids to whitelist from preventing integration (default `none`) |
 | -w   | --whitelist       | Vulnerable modules to whitelist from preventing integration (default `none`)      |
+|      | --settings        | Path to JSON config file                                                          |
```

Comment:
Just wondering if `config` or `configpath` would be a better argument name?

Reply:
I was wondering between `config` and `settings` for a while. In this submission I went with `settings` primarily because the Yargs example had `settings`: https://github.com/yargs/yargs/blob/master/docs/api.md#configobject. However, upon further review they even have `config` as the default: _If invoked without parameters, `.config()` will make `--config` the option to pass the JSON config file._

I will resubmit this PR with `config` instead of `settings`.